lib/ability.rb

```diff
@@ -148,7 +148,7 @@ def initialize(user, resource, controller = nil)
 
         can :index, User if controller.is_a?(FacilityUserReservationsController)
 
-        can [:list, :show], Facility
+        can [:list, :dashboard, :show], Facility
```

Comment:
Just curious, without this, what happens if you're a lowly facility staff and you try to get in to the manage facility? I would expect it to fail. Maybe we should add something in a feature spec?